repobee_sanitizer/_sanitize_repo.py

```diff
@@ -102,6 +107,42 @@ def create_extension_command(self) -> plug.ExtensionCommand:
         )
 
 
+def _parse_file_list(file_list) -> List[pathlib.Path]:
```

Comment:
It's a bit odd that the return value is type hinted, but not the argument. It's not significant enough that I'd require a change, but it looks odd.